benchmarks/bench_mnist.py

```diff
@@ -92,6 +92,8 @@ def load_data(dtype=np.float32, order='F'):
     'SampledRBF-SVM': make_pipeline(
         RBFSampler(gamma=0.015, n_components=1000), LinearSVC(C=100)),
     'LinearRegression-SAG': LogisticRegression(solver='sag', tol=1e-1, C=1e4),
+    'LinearRegression-SAGA': LogisticRegression(solver='saga', tol=1e-1,
+                                                C=1e4),
```

Comment:
Those two should be renamed to "LogisticRegression". Linear regression should be reserved for the ridge, huber of absolute losses.